src/utils/make-reg-exp.js

```diff
@@ -1,5 +1,17 @@
-import { escapeRegExp as escapeRe } from 'lodash';
+import escapeRegExp from 'lodash/escapeRegExp';
 
-export default function makeRegExp (str) {
-    return typeof str === 'string' ? new RegExp(escapeRe(str)) : str;
+export function parseRegExpString (regExp) {
+    if (typeof regExp !== 'string')
+        return regExp;
+
+    const containFlags = regExp.match(/^\/(.*?)\/([gim]*)$/);
+
+    if (containFlags)
```

Comment:
I guess this conditional logic is excess: we always can use the `return makeRegExp(containFlags[1], containFlags[2]);` expression.